the-9-exercises/src/App.jsx

```diff
@@ -1,14 +1,10 @@
-
-import './App.css'
-
-function App() {
-
-
+import Button1 from "./components/Button1";
```

Comment:
I would suggest you to change this component naming into something meaning because when you have many buttons naming them Button 1, 2, 3.... would confuse you as the project gets bigger.